libraries/core_libs/network/src/tarcap/packets_handlers/common/ext_votes_packet_handler.cpp

```diff
@@ -12,7 +12,7 @@ ExtVotesPacketHandler::ExtVotesPacketHandler(std::shared_ptr<PeersState> peers_s
 void ExtVotesPacketHandler::onNewPbftVote(std::shared_ptr<Vote> const &vote) {
   std::vector<dev::p2p::NodeID> peers_to_send;
   for (auto const &peer : peers_state_->getAllPeers()) {
-    if (!peer.second->isVoteKnown(vote->getHash())) {
+    if (!peer.second->isVoteKnown(vote->getHash()) && !peer.second->syncing_) {
```

Comment:
I think the peer.second->syncing_ is both for request blocks from peers and sending out blocks to peers.
For example, if a bootstrapping node is syncing from CN0, then other consensus nodes will not send votes to CN0. That's not right. In that case, we don't want sending votes to the bootstrapping node, but we still want sending votes to CN0

Reply:
I do not understand your example, why other node would not send votes to CN0? CN0 not syncing anymore it will receive votes as usual.   `syncing_` variable is set base on your status packet, where node that is sending that packet specifies if it is syncing or not.

Reply:
If node1 is syncing from CN0, the "syncing_" for node1 is true. How about the "syncing_" for CN0? True or false?

Reply:
it is false, if CN0 is up to date